testing/tests/DevExpress.ui.widgets.editors/textEditorParts/caretWorkaround.js

```diff
@@ -1,17 +1,13 @@
 import browser from 'core/utils/browser';
-import devices from 'core/devices';
 import caret from 'ui/text_box/utils.caret';
 
 export default function($input) {
     // NOTE: We decided to return to this problem when next Edge is released
     // Edge sets the caret to the end of the mask editor when iframe is used and focus is triggered by the focus() method
-    const isEdge = browser.msie && browser.version >= 15 && browser.version < 19;
```

Comment:
We should also remove the note if we are sure that the problem is not actual for chromium edge